db/migrate/20200528113707_add_search_index_to_facility.rb

```diff
@@ -0,0 +1,11 @@
+class AddSearchIndexToFacility < ActiveRecord::Migration[5.2]
+  def up
+    execute "CREATE INDEX index_gin_facilities_on_name ON facilities USING GIN ((to_tsvector('simple'::regconfig, COALESCE((name)::text, ''::text))));"
+    execute "CREATE INDEX index_gin_facilities_on_slug ON facilities USING GIN ((to_tsvector('simple'::regconfig, COALESCE((slug)::text, ''::text))));"
```

Comment:
These search indexes, do we need to worry about them affecting DB performance when run?

Reply:
Not yet, the users/facility tables are tiny, we needn't worry about this for now. The `GIN` indexes can get big, but don't have to worry about that now either, the `users` and `facilities` tables are both under 1mb right now on production.